# The Sing Test 

The Sign Test in statistics is a non-parametric test used to assess if the median of a distribution significantly differs from a given value, without making assumptions about the distribution shape.

#### 13.3.1
A random sample of 15 student nurses was given a test to measure their level of authoritarianism with
the following results:

Test at the .05 level of significance, the null hypothesis that 
the median score for the sampled population is 100. Determine 
the p value.

In [1]:
library(BSDA)


data <- data.frame(
    num = c(1:15),
    Authoritarianism = c(75,90,85,110,115,95,132,74,82,104,88,124,110,76,98)
)
attach(data)

SIGN.test(Authoritarianism, md=100, alternative = "two.sided", conf.level = 0.95)


Loading required package: lattice


Attaching package: ‘BSDA’


The following object is masked from ‘package:datasets’:

    Orange





	One-sample Sign-Test

data:  Authoritarianism
s = 6, p-value = 0.6072
alternative hypothesis: true median is not equal to 100
95 percent confidence interval:
  82.53451 110.00000
sample estimates:
median of x 
         95 

Achieved and Interpolated Confidence Intervals: 

                  Conf.Level  L.E.pt U.E.pt
Lower Achieved CI     0.8815 85.0000    110
Interpolated CI       0.9500 82.5345    110
Upper Achieved CI     0.9648 82.0000    110


Based on the p-value of 0.6072, which is greater than the significance 
level of 0.05, we fail to reject the null hypothesis. This means that 
there is not enough evidence to conclude that the median score for the 
sampled population is significantly different from 100.

#### 13.3.2

Determining the effects of grapefruit juice on pharmacokinetics of oral digoxin (a drug often
prescribed for heart ailments) was the goal of a study by Parker et al. (A-1). Seven healthy
nonsmoking volunteers participated in the study. Subjects took digoxin with water for 2 weeks,
no digoxin for 2 weeks, and digoxin with grapefruit juice for 2 weeks. The average peak plasma
digoxin concentration (Cmax) when subjects took digoxin with water is given in the first column of
the following table. The second column gives the Cmax concentration when subjects took digoxin
with grapefruit juice. May we conclude on the basis of these data that the Cmax concentration is
higher when digoxin is taken with grapefruit juice? Let $\alpha$ = .5.

In [2]:
data <- data.frame(
    h2o <- c(2.34,2.46,1.87,3.09,5.59,4.05,6.21),
    GFC <- c(3.03,3.46,1.97,3.81,3.07,2.62,3.44)
)

attach(data)


# Perform the sign test
SIGN.test(GFC, h2o, alternative = "greater", conf.level = 0.95)


	Dependent-samples Sign-Test

data:  GFC and h2o
S = 4, p-value = 0.5
alternative hypothesis: true median difference is greater than 0
95 percent confidence interval:
 -2.577143       Inf
sample estimates:
median of x-y 
          0.1 

Achieved and Interpolated Confidence Intervals: 

                  Conf.Level  L.E.pt U.E.pt
Lower Achieved CI     0.9375 -2.5200    Inf
Interpolated CI       0.9500 -2.5771    Inf
Upper Achieved CI     0.9922 -2.7700    Inf


Based on the p-value of 0.5, which is greater than the significance level of 0.05, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that the Cmax concentration is significantly higher when digoxin is taken with grapefruit juice based on the given data.

#### 13.3.3

A sample of 15 patients suffering from asthma participated in an experiment to study the effect of a
new treatment on pulmonary function. Among the various measurements recorded were those of
forced expiratory volume (liters) in 1 second (FEV1) before and after application of the treatment.
The results were as follows:

On the basis of these data, can one conclude that the treatment is effective in increasing the FEV1
level? Let $\alpha$ = .05 and find the p value

In [3]:
data <- data.frame(
    before <- c(1.69,2.77,1.00,1.66,3.00,.85,1.42,2.82,2.58,1.84,1.89,1.91,1.75,2.46,2.35),
    after <- c(1.69,2.22,3.07,3.35,3.00,2.74,3.61,5.14,2.44,4.17,2.42,2.94,3.04,4.62,4.42)
)
attach(data)

SIGN.test(before, after, alternative = "greater", conf.level = 0.95)


	Dependent-samples Sign-Test

data:  before and after
S = 2, p-value = 0.9983
alternative hypothesis: true median difference is greater than 0
95 percent confidence interval:
 -2.089952       Inf
sample estimates:
median of x-y 
        -1.69 

Achieved and Interpolated Confidence Intervals: 

                  Conf.Level L.E.pt U.E.pt
Lower Achieved CI     0.9408  -2.07    Inf
Interpolated CI       0.9500  -2.09    Inf
Upper Achieved CI     0.9824  -2.16    Inf


Based on the p-value of 0.9983, which is greater than the significance level of 0.05, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that the treatment is effective in increasing the FEV1 level based on the given data.